kibbeh/src/modules/room/chat/RoomChatInput.tsx

```diff
@@ -109,7 +109,7 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({ users }) => {
           }}
         />
       </div>
-      <div className="flex items-stretch">
+      <div className="flex flex items-stretch">
```

Comment:
🤔

Reply:
I just fixed this duplicate class. must have been a mistake